runtime/src/main/java/org/corfudb/runtime/object/CorfuCompileProxy.java

```diff
@@ -164,7 +166,9 @@ public CorfuCompileProxy(CorfuRuntime rt, UUID streamID, Class<T> type, Object[]
     public <R> R access(ICorfuSMRAccess<R, T> accessMethod,
                         Object[] conflictObject) {
         try (Timer.Context context = MetricsUtils.getConditionalContext(timerAccess)) {
-            return accessInner(accessMethod, conflictObject);
+            try (Timer.Context streamContext = MetricsUtils.getConditionalContext(timerAccessPerStream)) {
+                return accessInner(accessMethod, conflictObject);
+            }
```

Comment:
Shouldn't this be enough? Why is it necessary to have another latency histogram inside the VLO ?